src/apply-fixes.js

```diff
@@ -0,0 +1,10 @@
+export function applyTSLintAllFixes(linterResults, source, tslint) {
```

Comment:
This is a lot simpler than I imagined this being...

Reply:
Yes, me too im suprised by simplicity :)